episodes/chap5_python_client.md

```diff
@@ -118,51 +433,307 @@ list(self, name: 'str' = None, country: 'int' = None, collaboration: 'int' = Non
         the previously setup collaboration is used. Default value is None
 
 ```
+This shows you that you can filter the list of organizations (among others) by name,
+country, and collaboration.  It is also possible to request documentation of a higher
+level method, for example `help(client.organization)` or even `help(client)`.:
+
+::: spoiler
+
+#### Online documentation
 
-From parameters like the ones described above, it is important to note that -for the sake of consistency- the client methods use identifiers rather than names. This implies that, for example, to filter the organizations that belong to a a given collaboration, you need to know the collaboration's identifier first.
+To view all `Client` functions and their arguments without using `help()` you can use
+the [official documentation (docs.vantage6.ai)](https://docs.vantage6.ai/en/main/function-docs/_autosummary/vantage6.client.Client.html#vantage6-client-client). Which is the same as the Python client's docstring.
 
-You can get these identifiers through the UI, or through the client. For example, if you need to know the identifiers of the collaborations you have access to, you can use the `list()` function of the `client.collaboration` resource:
+Make sure you are viewing the documentation of the version of the client you are using.
+You can find the version of the client by one of the following commands:
 
 ```Python
-# Get all the details of all the collaborations you have access to:
-client.collaboration.list()
+import vantage6.client
+print(vantage6.client.__version__)
+```
 
-# Alternatively, to show only the 'id' and 'name' of each collaboration:
-client.collaboration.list(fields=['id', 'name'])
+or by running the following command in the terminal:
+
+```bash
+pip show vantage6-client | grep Version
```

Comment:
Would be good to mention command also for Windows terminal users - that should be `pip show vantage6-client | findstr Version`.